Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -167,27 +193,34 @@ void AdvancedPane::CreateLayout()
   mem1_override_slider_layout->setContentsMargins(0, 0, 0, 0);
   ram_override_layout->addLayout(mem1_override_slider_layout);
 
-  m_mem1_override_slider = new QSlider(Qt::Horizontal);
-  m_mem1_override_slider->setRange(24, 64);
+  m_mem1_override_slider = new ConfigSliderU32(24, 64, Config::MAIN_MEM1_SIZE, 0x100000);
   mem1_override_slider_layout->addWidget(m_mem1_override_slider);
 
-  m_mem1_override_slider_label = new QLabel();
-  mem1_override_slider_layout->addWidget(m_mem1_override_slider_label);
+  m_mem1_label =
+      new QLabel(tr("%1 MB (MEM1)").arg(QString::number(m_mem1_override_slider->value())));
+  mem1_override_slider_layout->addWidget(m_mem1_label);
+  connect(m_mem1_override_slider, &ConfigSliderU32::valueChanged, this, [this](int value) {
```

Comment:
I'm not sure which of these is right but you use `ConfigSliderU32::valueChanged` here but `QSlider::valueChanged` for the other sliders. Should probably be consistent?

Reply:
oops. Either way sends the same signal, but I tend to use QSlider because that's the class that defines the signal.